tutorials/Binary_Contingency_Scores.ipynb

```diff
@@ -0,0 +1,2738 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "18f07086-30d0-43ee-b560-f894b00508f3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import scores.categorical\n",
+    "import xarray as xr"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7e3b4f1c-7e82-4b29-9353-3c52eae9fa1e",
+   "metadata": {
+    "jp-MarkdownHeadingCollapsed": true
+   },
+   "source": [
+    "## Overview of Binary Contingency Scores\n",
+    "\n",
+    "Many people will be familiar with the following terms: true positives, false negatives, true positive rate. These terms and many more all apply to a binary contingency table. Producing these categorical statistics is inherently a three-step process.\n",
```

Comment:
I'm okay with the contingency table class using "true positive" etc, but it would be good to define what they mean for people from the met community who are unfamiliar with these terms.

Reply:
Updated.